### PR TITLE
Added new Example to Edb Library

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by svandenb-dev — APPROVED** [review]

File: examples/00-EDB/09_Configuration.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+                                  comp_type_col=2,
+                                  value_col=3)
+
+###############################################################################
+# Import Definitions
+# ~~~~~~~~~~~~~~~~~~
+# A definitions file is a json containing, for each part name the model associated.
+# Model can be RLC, Sparameter or Spice.
+# Once imported the definition is applied to the board.
+# Json file is store for convenience in aedb folder.
```
Comment (MaxJPRey) [inline examples/00-EDB/09_Configuration.py:65]:
```suggestion
# Json file is stored for convenience in aedb folder.
```

File: examples/00-EDB/09_Configuration.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+# ~~~~~~~~
+edbapp.save_edb()
+
+###############################################################################
+# Configure Setup
+# ~~~~~~~~~~~~~~~
+# This step allows to define the project. It includes:
+#  - Definition of nets to be included into the cutout
+#  - Cutout details
+#  - Components on which create the ports
```
Comment (MaxJPRey) [inline examples/00-EDB/09_Configuration.py:99]:
```suggestion
#  - Components on which to create the ports
```

File: examples/00-EDB/09_Configuration.py [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+###############################################################################
+# Get Results
+# ~~~~~~~~~~~
+# S Parameter data will be loaded at the end of simulation.
+solutions = h3d.post.get_solution_data()
+
+###############################################################################
+# Plot Results
+# ~~~~~~~~~~~~
+# S Parameter data will be plotted at the end of simulation.
```
Comment (MaxJPRey) [inline examples/00-EDB/09_Configuration.py:157]:
```suggestion
# Plot S Parameter.
```

File: examples/00-EDB/09_Configuration.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+solutions.plot(solutions.expressions, "db20")
+
+###############################################################################
+# Save and Close AEDT
+# ~~~~~~~~~~~~~~~~~~~
+# S Parameter data will be plotted at the end of simulation.
```
Comment (MaxJPRey) [inline examples/00-EDB/09_Configuration.py:163]: I think we could change this comment because the S-Parameters were displayed in the previous step.

**Review by MaxJPRey — APPROVED** [review]

**Review by ring630 — COMMENTED** [review]
It is suggested to import Comp Definition before BOM.

**Pull request merged**